templates/zerver/emails/digest.html

```diff
@@ -2,6 +2,72 @@
 Mail sent to a user who hasn't logged in for 24 hours.
 #}
 
+<span style="display:none">
```

Comment:
We should instead put the CSS at https://www.emailonacid.com/blog/article/email-development/tips-for-coding-email-preheaders into a class, and put the class definition into email_base.html. (Or maybe better, add a {% block preheader %} to the email_base template.)